commercetools/commercetools-sdk-java-importapi/src/main/java-generated/com/commercetools/importapi/models/importsummaries/OperationStatesBuilder.java

```diff
@@ -9,75 +9,89 @@
 @Generated(value = "io.vrap.rmf.codegen.rendring.CoreCodeGenerator", comments = "https://github.com/vrapio/rmf-codegen")
 public final class OperationStatesBuilder implements Builder<OperationStates> {
 
-    private Long ValidationFailed;
+    private Long processing;
 
-    private Long Unresolved;
+    private Long validationFailed;
 
-    private Long WaitForMasterVariant;
+    private Long unresolved;
 
-    private Long Imported;
+    private Long waitForMasterVariant;
 
-    private Long Rejected;
+    private Long imported;
 
-    public OperationStatesBuilder ValidationFailed(final Long ValidationFailed) {
-        this.ValidationFailed = ValidationFailed;
+    private Long rejected;
+
+    public OperationStatesBuilder processing(final Long processing) {
+        this.processing = processing;
+        return this;
+    }
+
+    public OperationStatesBuilder validationFailed(final Long validationFailed) {
+        this.validationFailed = validationFailed;
         return this;
     }
 
-    public OperationStatesBuilder Unresolved(final Long Unresolved) {
-        this.Unresolved = Unresolved;
+    public OperationStatesBuilder unresolved(final Long unresolved) {
```

Comment:
breaking: methods are now lower cased

Reply:
yep pretty cool, it were announced to customers